xml/System.Numerics/BigInteger.xml

```diff
@@ -3036,10 +3218,31 @@ value Mod 2 = 0
         <Parameter Name="value" Type="System.Numerics.BigInteger" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">The value to convert to a single-precision floating-point value.</param>
+        <summary>Defines an explicit conversion of a <see cref="T:System.Numerics.BigInteger" /> object to a single-precision floating-point value.</summary>
+        <returns>An object that contains the closest possible representation of the <paramref name="value" /> parameter.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The overloads of the <xref:System.Numerics.BigInteger.op_Explicit(System.Decimal)~System.Numerics.BigInteger> method define the types to which or from which a <xref:System.Numerics.BigInteger> object can be converted. Language compilers do not perform this conversion automatically because it can involve data loss or a loss of precision. Instead, they perform the conversion only if a casting operator (in C#) or a conversion function (such as `CType` or `CSng` in Visual Basic) is used. Otherwise, they display a compiler error.    
+
+ Because the <xref:System.Numerics.BigInteger> value can be outside the range of the <xref:System.Single> data type, this operation is a narrowing conversion. If the conversion is unsuccessful, it does not throw an <xref:System.OverflowException>. Instead, if the <xref:System.Numerics.BigInteger> value is less than <xref:System.Single.MinValue>, the resulting <xref:System.Single> value is <xref:System.Single.NegativeInfinity>. If the <xref:System.Numerics.BigInteger> value is greater than <xref:System.Single.MaxValue>, the resulting <xref:System.Single> value is <xref:System.Single.PositiveInfinity>.
```

Comment:
The xrefs to Single.MinValue, Single.MaxValue, Single.PositiveInfinity, and Single.NegativeInfinity should all be `?displayProperty=fullName`.